components/admin-panel/sections/accounting/CreateAccountingCategoryModal.tsx

```diff
@@ -55,15 +55,15 @@ export function CreateAccountingCategoryModal(props: CreateAccountingCategoryMod
   });
 
   return (
-    <StyledModal onClose={props.onClose}>
-      <form onSubmit={e => formik.handleSubmit(e)}>
-        <ModalHeader>
-          <FormattedMessage defaultMessage="Create accounting category" />
-        </ModalHeader>
-        <ModalBody>
-          <AccountingCategoryForm formik={formik} />
-        </ModalBody>
-        <ModalFooter showDivider={false}>
+    <StyledModal onClose={props.onClose} hasUnsavedChanges={formik.dirty}>
```

Comment:
@hdiniz also added this small prop to show a confirm before closing the modal if the form is not empty.